resources/EPvalidator.rb

```diff
@@ -410,6 +410,8 @@ def self.run_validator(hpxml_doc)
         "[WaterHeaterType='storage water heater' or WaterHeaterType='instantaneous water heater' or WaterHeaterType='heat pump water heater' or WaterHeaterType='space-heating boiler with storage tank' or WaterHeaterType='space-heating boiler with tankless coil']" => one, # See [WHType=Tank] or [WHType=Tankless] or [WHType=HeatPump] or [WHType=Indirect] or [WHType=CombiTankless]
         "[Location='living space' or Location='basement - unconditioned' or Location='basement - conditioned' or Location='attic - unvented' or Location='attic - vented' or Location='garage' or Location='crawlspace - unvented' or Location='crawlspace - vented']" => one,
         "FractionDHWLoadServed" => one,
+        "[EnergyFactor | UniformEnergyFactor]" => one,
```

Comment:
Was this an accident?

Reply:
Oops! Must be the conflicts I solved today, yeah for sure it's an error!